### PR TITLE
configure intellij gradle plugin for publish task

### PR DESCRIPTION
**Pull request by patflynn**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -30,6 +30,12 @@ subprojects {
         version = ideaVersion
         downloadSources = true
         sandboxDirectory = "${rootProject.buildDir}/idea-sandbox"
+
+        publish {
+            username System.getenv('IJ_REPO_USERNAME')
+            password System.getenv('IJ_REPO_PASSWORD')
```
Comment (loosebazooka) [inline build.gradle]: not sure I like this.

Comment (loosebazooka) [inline build.gradle]: you dont want to read this from stdin?

Comment (loosebazooka) [inline build.gradle]: oh it's from travis

**Comment by loosebazooka** [comment]
lgtm

**Comment by patflynn** [comment]
#236

**Pull request merged**